Topic edits (#98)
HCL-TECH-SOFTWARE/digital-experience: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_web_content_dev.md

10.  Log in to HCL Portal to verify that your portal server works.

## Enabling federated security

After you install your web experience, a default file-based repository is the available user registry. Depending on your requirements, you might have to enable a federated LDAP user registry.

!!! note 
    If you set **Use Administrator IDs stored in your LDAP user registry** to ``yes``, the WebSphere® Application Server and HCL Portal user IDs and passwords are changed to the LDAP user IDs and passwords. If you do not want to change both user IDs and passwords to match the LDAP user ID and password, set this value to ``no``. After you configure the LDAP user registry, you can manually change the user IDs and passwords.

To enable federated security:
1.  Get the latest updates for the wizard, and apply the most recent CF. For more information about applying the latest fix pack, see [Apply Combined Cumulative Fix](../../../../../deployment/install/traditional/cf_install/index.md)for more topic information.

Skip the next step, if you applied the most recent fix pack.

2.  Open the Configuration wizard. Go to http://your_server:10200/hcl/wizard.

> If you are working with HCL Digital Experience V8.5 or 9 software at a level prior to CF18, the wizard address is  http://*your_server*:10200/ibm/wizard. After installing CF18, the configuration wizard  automatically adjusts to http://*your_server*:10200/hcl/wizard.

Reviewer: ```suggestion
If you are working with HCL Digital Experience V8.5 or 9 software at a level prior to CF18, the wizard address is http://*your_server*:10200/ibm/wizard. After installing CF18, the configuration wizard  automatically adjusts to http://*your_server*:10200/hcl/wizard.
```

Author: Yes

There is a known issue with Chrome version 45.x and the Configuration Wizard. If you are experiencing difficulties, use a different browser when you access the wizard.

3.  Log in to the Configuration wizard with the administrative ID for the configuration wizard profile, cw_profile.

> If your specified language is not  supported for the user interface, you might see the English version. For details about supported languages and the language codes for all of the HCL Portal user interfaces, see [Language support](../../../../../deployment/manage/portal_admin_tools/language_support/index.md) in the HCL Digital Experience Version 8.5 documentation.

Reviewer: ```suggestion
If your specified language is not supported for the user interface, you might see the English version. For details about supported languages and the language codes for all of the HCL Portal user interfaces, see [Language support](../../../../../deployment/manage/portal_admin_tools/language_support/index.md) in the HCL Digital Experience Version 8.5 documentation.
```

Author: Yes

    -   Click **Download Files** to run the steps remotely.
    -   Click **Run All Steps** to run the steps locally.
    

> 8.  Log in to HCL Portal to verify that your portal server workings.

Reviewer: ```suggestion
8.  Log in to HCL Portal to verify that your portal server works.
```

Author: Yes

Use syndication to synchronize content between authoring, staging, and publishing environments.

1.  Plan your syndication strategies. See [Syndication relationships](../../../../../manage_content/wcm_delivery/syndication/wcm_syndication_overview.md).

2.  Define syndication properties. See [Syndication properties](../../../../../manage_content/wcm_delivery/syndication/wcm_config_prop_syndication.md).

> 3.  Tune your syndication strategy to improve performance. See [Syndication tuning](../../../../../manage_content/wcm_delivery/syndication/wcm_syndication_tuning.md)

Reviewer: ```suggestion
3.  Tune your syndication strategy to improve performance. See [Syndication tuning](../../../../../manage_content/wcm_delivery/syndication/wcm_syndication_tuning.md).
```

Author: Yes

> 4.  Create your syndication relationships. See [Creating a syndication relationship by using the Administration Portlet](../../../../../manage_content/wcm_delivery/syndication/manage_synd_subs/wcm_syndication_settingup.md)

Reviewer: ```suggestion
4.  Create your syndication relationships. See [Creating a syndication relationship by using the Administration Portlet](../../../../../manage_content/wcm_delivery/syndication/manage_synd_subs/wcm_syndication_settingup.md).
```

Author: Yes

## Deploying the delivery environment

The delivery environment is used to deliver your website to website viewers. The delivery environment is deployed based on the requirements that you define in the project design document.

1.  Based on the database architecture that is defined in the project design document, the database administrator completes the following steps:

    1.  Deploys a database server for the delivery environment.

    2.  Clones the data that is stored on the authoring database onto the delivery database.

2.  Based on the server architecture that is defined in the project design document, the HCL Portal administrator completes the following steps:

    1.  Installs an HCL Portal server or cluster of servers.

>     2.  Configures the HCL Portal server or cluster to use the database server the database administrator set up.

Reviewer: ```suggestion
    2.  Configures the HCL Portal server or cluster to use the database server the database administrator setup.
```

Author: Yes

    3.  Configures various WebSphere Application Server, HCL Portal and Web Content Manager configuration properties to ensure that the system is correctly set up for web content delivery and is tuned for optimal performance.

3.  Based on the information architecture and security architecture that is defined in the project design document, the HCL Portal administrator completes the following steps:

    1.  Creates all pages that are required by the web content system.

    2.  Adds all required web content viewer portlets to the appropriate pages.

4.  The HCL Portal administrator configures and enables syndication.

5.  All administrators complete final testing and tuning of the delivery environment.

The delivery environment is ready to use.

## Going live with your website

> After your environments are installed, the authoring system is complete, your default content is created, and the system is fully, you are ready to go live.

Reviewer: ```suggestion
After your environments are installed, the authoring system is complete, your default content is created, and the system is tested, you are ready to go live.
```

Author: Yes
